src/components/main.vue

```diff
@@ -32,25 +34,23 @@
 
 <script lang="ts">
   import { LocationProvider } from '@empathyco/x-components';
-  import Vue from 'vue';
   import { Component } from 'vue-property-decorator';
-  import Redirection from './results/redirection.vue';
-  import NoResultsMessage from './no-results-message.vue';
+  import Redirection from './search-has-query/redirection.vue';
   import PartialResults from './results/partial-results.vue';
   import Recommendations from './results/recommendations.vue';
   import Results from './results/results.vue';
-  import SpellcheckMessage from './spellcheck-message.vue';
+  import HasQueryMixin from './has-query.mixin.vue';
 
   @Component({
     components: {
       LocationProvider,
-      NoResultsMessage,
       PartialResults,
       Recommendations,
       Redirection,
       Results,
```

Comment:
I agree with both `PartialResults` and `Redirection`, but I'm afraid `Results` can't be lazy loaded and should go in the main chunk, as their included in the `Top Clicked`, which appears in the initial render.

Reply:
I don't get you, top clicked are rendered using the `Recommendations` component, which uses the `Result` (notice it's singular). Both of `Recommendations` and `Result` components are included in the `x-modal` chunk which is fine for me... `Results` (which I would probably rename to `ResultsGrid` or with any other suffix to avoid misunderstandings) Is rendered later in the "search" chunk, just when there is a search available, uses the `Result` from the previous `x-modal` chunk.

![image](https://user-images.githubusercontent.com/68222542/152303084-a1e98e64-4397-4a02-a322-df504f63fbda.png)
![image](https://user-images.githubusercontent.com/68222542/152303148-2d7a3a4f-7fdc-4d56-a711-3a3c97a32433.png)

Reply:
I read `Result` instead of `Results` 🤦🏿 , my bad